<html xmlns:string="xalan://java.lang.String" xmlns:lxslt="http://xml.apache.org/xslt">
<head>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>Standard Error from JUnitTclTest</title>
</head>
<body>
<pre>ptolemy.kernel.util.InternalErrorException: Stdout-0 IOExeception throwed.
  in .DistributedReceiver1.HlaManager
Because:
Stream closed
	at org.hlacerti.lib.CertiRtig$_StreamReaderThread._read(CertiRtig.java:405)
	at org.hlacerti.lib.CertiRtig$_StreamReaderThread.run(CertiRtig.java:374)
Caused by: java.io.IOException: Stream closed
	at java.io.BufferedInputStream.getBufIfOpen(BufferedInputStream.java:170)
	at java.io.BufferedInputStream.read1(BufferedInputStream.java:283)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:345)
	at sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:284)
	at sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:326)
	at sun.nio.cs.StreamDecoder.read(StreamDecoder.java:178)
	at java.io.InputStreamReader.read(InputStreamReader.java:184)
	at org.hlacerti.lib.CertiRtig$_StreamReaderThread._read(CertiRtig.java:394)
	... 1 more
Caused by: java.io.IOException: Stream closed
	at java.io.BufferedInputStream.getBufIfOpen(BufferedInputStream.java:170)
	at java.io.BufferedInputStream.read1(BufferedInputStream.java:283)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:345)
	at sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:284)
	at sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:326)
	at sun.nio.cs.StreamDecoder.read(StreamDecoder.java:178)
	at java.io.InputStreamReader.read(InputStreamReader.java:184)
	at org.hlacerti.lib.CertiRtig$_StreamReaderThread._read(CertiRtig.java:394)
	at org.hlacerti.lib.CertiRtig$_StreamReaderThread.run(CertiRtig.java:374)
ptolemy.kernel.util.IllegalActionException: RTIG could not open FED file: StaticMultiInstance222.fed
  in .DistributedReceiver1.HlaManager
Because:
Module not found
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:747)
	at ptolemy.actor.CompositeActor.initialize(CompositeActor.java:912)
	at ptolemy.actor.Manager.initialize(Manager.java:724)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.lib.hoc.ModelReference$1.run(ModelReference.java:626)
Caused by: hla.rti.CouldNotOpenFED: Module not found serial:0
	at certi.rti.impl.CertiRtiAmbassador.translateException(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.processRequest(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.createFederationExecution(Unknown Source)
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:734)
	... 4 more
Caused by: hla.rti.CouldNotOpenFED: Module not found serial:0
	at certi.rti.impl.CertiRtiAmbassador.translateException(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.processRequest(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.createFederationExecution(Unknown Source)
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:734)
	at ptolemy.actor.CompositeActor.initialize(CompositeActor.java:912)
	at ptolemy.actor.Manager.initialize(Manager.java:724)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.lib.hoc.ModelReference$1.run(ModelReference.java:626)
java.lang.RuntimeException: ptolemy.kernel.util.IllegalActionException: RTIG could not open FED file: StaticMultiInstance222.fed
  in .DistributedReceiver1.HlaManager
Because:
Module not found
	at ptolemy.util.MessageHandler.error(MessageHandler.java:115)
	at ptolemy.actor.lib.hoc.ModelReference.executionError(ModelReference.java:450)
	at ptolemy.actor.Manager$ErrorReportingThread.run(Manager.java:1890)
Caused by: ptolemy.kernel.util.IllegalActionException: RTIG could not open FED file: StaticMultiInstance222.fed
  in .DistributedReceiver1.HlaManager
Because:
Module not found
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:747)
	at ptolemy.actor.CompositeActor.initialize(CompositeActor.java:912)
	at ptolemy.actor.Manager.initialize(Manager.java:724)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.lib.hoc.ModelReference$1.run(ModelReference.java:626)
Caused by: hla.rti.CouldNotOpenFED: Module not found serial:0
	at certi.rti.impl.CertiRtiAmbassador.translateException(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.processRequest(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.createFederationExecution(Unknown Source)
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:734)
	... 4 more
java.lang.RuntimeException: Execution failed
	at ptolemy.moml.MoMLSimpleApplication$UnloadThread.run(MoMLSimpleApplication.java:402)
Caused by: ptolemy.kernel.util.IllegalActionException: Execution failed.
  in .2Classes2Values2Instances.DistributedSender1
Because:
RTIG could not open FED file: StaticMultiInstance222.fed
  in .DistributedSender1.HlaManager
Because:
Module not found
	at ptolemy.actor.lib.hoc.ModelReference.fire(ModelReference.java:588)
	at ptolemy.vergil.actor.lib.VisualModelReference.fire(VisualModelReference.java:414)
	at ptolemy.actor.AtomicActor.iterate(AtomicActor.java:512)
	at ptolemy.actor.Director.fire(Director.java:360)
	at ptolemy.actor.CompositeActor.fire(CompositeActor.java:455)
	at ptolemy.actor.Manager.iterate(Manager.java:851)
	at ptolemy.actor.Manager.execute(Manager.java:369)
	at ptolemy.actor.Manager.run(Manager.java:1263)
	at ptolemy.actor.Manager$PtolemyRunThread.run(Manager.java:1915)
Caused by: ptolemy.kernel.util.IllegalActionException: RTIG could not open FED file: StaticMultiInstance222.fed
  in .DistributedSender1.HlaManager
Because:
Module not found
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:747)
	at ptolemy.actor.CompositeActor.initialize(CompositeActor.java:912)
	at ptolemy.actor.Manager.initialize(Manager.java:724)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.lib.hoc.ModelReference.fire(ModelReference.java:586)
	... 8 more
Caused by: hla.rti.CouldNotOpenFED: Module not found serial:0
	at certi.rti.impl.CertiRtiAmbassador.translateException(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.processRequest(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.createFederationExecution(Unknown Source)
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:734)
	... 12 more
java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at ptolemy.util.test.junit.AutoKnownFailedTests.RunModel(AutoKnownFailedTests.java:105)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at junitparams.internal.InvokeParameterisedMethod.evaluate(InvokeParameterisedMethod.java:102)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
Caused by: ptolemy.kernel.util.IllegalActionException: Execution failed.
  in .2Classes2Values2Instances.DistributedSender1
Because:
RTIG could not open FED file: StaticMultiInstance222.fed
  in .DistributedSender1.HlaManager
Because:
Module not found
	at ptolemy.actor.lib.hoc.ModelReference.fire(ModelReference.java:588)
	at ptolemy.vergil.actor.lib.VisualModelReference.fire(VisualModelReference.java:414)
	at ptolemy.actor.AtomicActor.iterate(AtomicActor.java:512)
	at ptolemy.actor.Director.fire(Director.java:360)
	at ptolemy.actor.CompositeActor.fire(CompositeActor.java:455)
	at ptolemy.actor.Manager.iterate(Manager.java:851)
	at ptolemy.actor.Manager.execute(Manager.java:369)
	at ptolemy.actor.Manager.run(Manager.java:1263)
	at ptolemy.actor.Manager$PtolemyRunThread.run(Manager.java:1915)
Caused by: ptolemy.kernel.util.IllegalActionException: RTIG could not open FED file: StaticMultiInstance222.fed
  in .DistributedSender1.HlaManager
Because:
Module not found
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:747)
	at ptolemy.actor.CompositeActor.initialize(CompositeActor.java:912)
	at ptolemy.actor.Manager.initialize(Manager.java:724)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.lib.hoc.ModelReference.fire(ModelReference.java:586)
	... 8 more
Caused by: hla.rti.CouldNotOpenFED: Module not found serial:0
	at certi.rti.impl.CertiRtiAmbassador.translateException(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.processRequest(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.createFederationExecution(Unknown Source)
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:734)
	... 12 more
java.lang.RuntimeException: Execution failed
	at ptolemy.moml.MoMLSimpleApplication$UnloadThread.run(MoMLSimpleApplication.java:402)
Caused by: ptolemy.kernel.util.IllegalActionException: Execution failed.
  in .2Values1Instance.DistributedSender1
Because:
RTIG could not open FED file: StaticMultiInstance21.fed
  in .DistributedSender1.HlaManager
Because:
Module not found
	at ptolemy.actor.lib.hoc.ModelReference.fire(ModelReference.java:588)
	at ptolemy.vergil.actor.lib.VisualModelReference.fire(VisualModelReference.java:414)
	at ptolemy.actor.AtomicActor.iterate(AtomicActor.java:512)
	at ptolemy.actor.Director.fire(Director.java:360)
	at ptolemy.actor.CompositeActor.fire(CompositeActor.java:455)
	at ptolemy.actor.Manager.iterate(Manager.java:851)
	at ptolemy.actor.Manager.execute(Manager.java:369)
	at ptolemy.actor.Manager.run(Manager.java:1263)
	at ptolemy.actor.Manager$PtolemyRunThread.run(Manager.java:1915)
Caused by: ptolemy.kernel.util.IllegalActionException: RTIG could not open FED file: StaticMultiInstance21.fed
  in .DistributedSender1.HlaManager
Because:
Module not found
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:747)
	at ptolemy.actor.CompositeActor.initialize(CompositeActor.java:912)
	at ptolemy.actor.Manager.initialize(Manager.java:724)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.lib.hoc.ModelReference.fire(ModelReference.java:586)
	... 8 more
Caused by: hla.rti.CouldNotOpenFED: Module not found serial:0
	at certi.rti.impl.CertiRtiAmbassador.translateException(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.processRequest(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.createFederationExecution(Unknown Source)
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:734)
	... 12 more
java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at ptolemy.util.test.junit.AutoKnownFailedTests.RunModel(AutoKnownFailedTests.java:105)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at junitparams.internal.InvokeParameterisedMethod.evaluate(InvokeParameterisedMethod.java:102)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
Caused by: ptolemy.kernel.util.IllegalActionException: Execution failed.
  in .2Values1Instance.DistributedSender1
Because:
RTIG could not open FED file: StaticMultiInstance21.fed
  in .DistributedSender1.HlaManager
Because:
Module not found
	at ptolemy.actor.lib.hoc.ModelReference.fire(ModelReference.java:588)
	at ptolemy.vergil.actor.lib.VisualModelReference.fire(VisualModelReference.java:414)
	at ptolemy.actor.AtomicActor.iterate(AtomicActor.java:512)
	at ptolemy.actor.Director.fire(Director.java:360)
	at ptolemy.actor.CompositeActor.fire(CompositeActor.java:455)
	at ptolemy.actor.Manager.iterate(Manager.java:851)
	at ptolemy.actor.Manager.execute(Manager.java:369)
	at ptolemy.actor.Manager.run(Manager.java:1263)
	at ptolemy.actor.Manager$PtolemyRunThread.run(Manager.java:1915)
Caused by: ptolemy.kernel.util.IllegalActionException: RTIG could not open FED file: StaticMultiInstance21.fed
  in .DistributedSender1.HlaManager
Because:
Module not found
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:747)
	at ptolemy.actor.CompositeActor.initialize(CompositeActor.java:912)
	at ptolemy.actor.Manager.initialize(Manager.java:724)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.lib.hoc.ModelReference.fire(ModelReference.java:586)
	... 8 more
Caused by: hla.rti.CouldNotOpenFED: Module not found serial:0
	at certi.rti.impl.CertiRtiAmbassador.translateException(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.processRequest(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.createFederationExecution(Unknown Source)
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:734)
	... 12 more
ptolemy.kernel.util.IllegalActionException: RTIG could not open FED file: StaticMultiInstance21.fed
  in .DistributedReceiver1.HlaManager
Because:
Module not found
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:747)
	at ptolemy.actor.CompositeActor.initialize(CompositeActor.java:912)
	at ptolemy.actor.Manager.initialize(Manager.java:724)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.lib.hoc.ModelReference$1.run(ModelReference.java:626)
Caused by: hla.rti.CouldNotOpenFED: Module not found serial:0
	at certi.rti.impl.CertiRtiAmbassador.translateException(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.processRequest(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.createFederationExecution(Unknown Source)
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:734)
	... 4 more
Caused by: hla.rti.CouldNotOpenFED: Module not found serial:0
	at certi.rti.impl.CertiRtiAmbassador.translateException(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.processRequest(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.createFederationExecution(Unknown Source)
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:734)
	at ptolemy.actor.CompositeActor.initialize(CompositeActor.java:912)
	at ptolemy.actor.Manager.initialize(Manager.java:724)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.lib.hoc.ModelReference$1.run(ModelReference.java:626)
java.lang.RuntimeException: ptolemy.kernel.util.IllegalActionException: RTIG could not open FED file: StaticMultiInstance21.fed
  in .DistributedReceiver1.HlaManager
Because:
Module not found
	at ptolemy.util.MessageHandler.error(MessageHandler.java:115)
	at ptolemy.actor.lib.hoc.ModelReference.executionError(ModelReference.java:450)
	at ptolemy.actor.Manager$ErrorReportingThread.run(Manager.java:1890)
Caused by: ptolemy.kernel.util.IllegalActionException: RTIG could not open FED file: StaticMultiInstance21.fed
  in .DistributedReceiver1.HlaManager
Because:
Module not found
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:747)
	at ptolemy.actor.CompositeActor.initialize(CompositeActor.java:912)
	at ptolemy.actor.Manager.initialize(Manager.java:724)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.lib.hoc.ModelReference$1.run(ModelReference.java:626)
Caused by: hla.rti.CouldNotOpenFED: Module not found serial:0
	at certi.rti.impl.CertiRtiAmbassador.translateException(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.processRequest(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.createFederationExecution(Unknown Source)
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:734)
	... 4 more
ptolemy.kernel.util.IllegalActionException: RTIG could not open FED file: StaticMultiInstance22.fed
  in .DistributedReceiver1.HlaManager
Because:
Module not found
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:747)
	at ptolemy.actor.CompositeActor.initialize(CompositeActor.java:912)
	at ptolemy.actor.Manager.initialize(Manager.java:724)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.lib.hoc.ModelReference$1.run(ModelReference.java:626)
Caused by: hla.rti.CouldNotOpenFED: Module not found serial:0
	at certi.rti.impl.CertiRtiAmbassador.translateException(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.processRequest(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.createFederationExecution(Unknown Source)
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:734)
	... 4 more
Caused by: hla.rti.CouldNotOpenFED: Module not found serial:0
	at certi.rti.impl.CertiRtiAmbassador.translateException(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.processRequest(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.createFederationExecution(Unknown Source)
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:734)
	at ptolemy.actor.CompositeActor.initialize(CompositeActor.java:912)
	at ptolemy.actor.Manager.initialize(Manager.java:724)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.lib.hoc.ModelReference$1.run(ModelReference.java:626)
java.lang.RuntimeException: ptolemy.kernel.util.IllegalActionException: RTIG could not open FED file: StaticMultiInstance22.fed
  in .DistributedReceiver1.HlaManager
Because:
Module not found
	at ptolemy.util.MessageHandler.error(MessageHandler.java:115)
	at ptolemy.actor.lib.hoc.ModelReference.executionError(ModelReference.java:450)
	at ptolemy.actor.Manager$ErrorReportingThread.run(Manager.java:1890)
Caused by: ptolemy.kernel.util.IllegalActionException: RTIG could not open FED file: StaticMultiInstance22.fed
  in .DistributedReceiver1.HlaManager
Because:
Module not found
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:747)
	at ptolemy.actor.CompositeActor.initialize(CompositeActor.java:912)
	at ptolemy.actor.Manager.initialize(Manager.java:724)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.lib.hoc.ModelReference$1.run(ModelReference.java:626)
Caused by: hla.rti.CouldNotOpenFED: Module not found serial:0
	at certi.rti.impl.CertiRtiAmbassador.translateException(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.processRequest(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.createFederationExecution(Unknown Source)
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:734)
	... 4 more
java.lang.RuntimeException: Execution failed
	at ptolemy.moml.MoMLSimpleApplication$UnloadThread.run(MoMLSimpleApplication.java:402)
Caused by: ptolemy.kernel.util.IllegalActionException: Execution failed.
  in .2Values2Instances.DistributedSender1
Because:
RTIG could not open FED file: StaticMultiInstance22.fed
  in .DistributedSender1.HlaManager
Because:
Module not found
	at ptolemy.actor.lib.hoc.ModelReference.fire(ModelReference.java:588)
	at ptolemy.vergil.actor.lib.VisualModelReference.fire(VisualModelReference.java:414)
	at ptolemy.actor.AtomicActor.iterate(AtomicActor.java:512)
	at ptolemy.actor.Director.fire(Director.java:360)
	at ptolemy.actor.CompositeActor.fire(CompositeActor.java:455)
	at ptolemy.actor.Manager.iterate(Manager.java:851)
	at ptolemy.actor.Manager.execute(Manager.java:369)
	at ptolemy.actor.Manager.run(Manager.java:1263)
	at ptolemy.actor.Manager$PtolemyRunThread.run(Manager.java:1915)
Caused by: ptolemy.kernel.util.IllegalActionException: RTIG could not open FED file: StaticMultiInstance22.fed
  in .DistributedSender1.HlaManager
Because:
Module not found
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:747)
	at ptolemy.actor.CompositeActor.initialize(CompositeActor.java:912)
	at ptolemy.actor.Manager.initialize(Manager.java:724)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.lib.hoc.ModelReference.fire(ModelReference.java:586)
	... 8 more
Caused by: hla.rti.CouldNotOpenFED: Module not found serial:0
	at certi.rti.impl.CertiRtiAmbassador.translateException(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.processRequest(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.createFederationExecution(Unknown Source)
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:734)
	... 12 more
java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at ptolemy.util.test.junit.AutoKnownFailedTests.RunModel(AutoKnownFailedTests.java:105)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at junitparams.internal.InvokeParameterisedMethod.evaluate(InvokeParameterisedMethod.java:102)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
Caused by: ptolemy.kernel.util.IllegalActionException: Execution failed.
  in .2Values2Instances.DistributedSender1
Because:
RTIG could not open FED file: StaticMultiInstance22.fed
  in .DistributedSender1.HlaManager
Because:
Module not found
	at ptolemy.actor.lib.hoc.ModelReference.fire(ModelReference.java:588)
	at ptolemy.vergil.actor.lib.VisualModelReference.fire(VisualModelReference.java:414)
	at ptolemy.actor.AtomicActor.iterate(AtomicActor.java:512)
	at ptolemy.actor.Director.fire(Director.java:360)
	at ptolemy.actor.CompositeActor.fire(CompositeActor.java:455)
	at ptolemy.actor.Manager.iterate(Manager.java:851)
	at ptolemy.actor.Manager.execute(Manager.java:369)
	at ptolemy.actor.Manager.run(Manager.java:1263)
	at ptolemy.actor.Manager$PtolemyRunThread.run(Manager.java:1915)
Caused by: ptolemy.kernel.util.IllegalActionException: RTIG could not open FED file: StaticMultiInstance22.fed
  in .DistributedSender1.HlaManager
Because:
Module not found
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:747)
	at ptolemy.actor.CompositeActor.initialize(CompositeActor.java:912)
	at ptolemy.actor.Manager.initialize(Manager.java:724)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.lib.hoc.ModelReference.fire(ModelReference.java:586)
	... 8 more
Caused by: hla.rti.CouldNotOpenFED: Module not found serial:0
	at certi.rti.impl.CertiRtiAmbassador.translateException(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.processRequest(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.createFederationExecution(Unknown Source)
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:734)
	... 12 more
ptolemy.kernel.util.IllegalActionException: RTIG could not open FED file: MultiDataTypes.fed
  in .MultiDataTypesConsumer.HlaManager
Because:
Module not found
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:747)
	at ptolemy.actor.CompositeActor.initialize(CompositeActor.java:912)
	at ptolemy.actor.Manager.initialize(Manager.java:724)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.lib.hoc.ModelReference$1.run(ModelReference.java:626)
Caused by: hla.rti.CouldNotOpenFED: Module not found serial:0
	at certi.rti.impl.CertiRtiAmbassador.translateException(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.processRequest(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.createFederationExecution(Unknown Source)
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:734)
	... 4 more
Caused by: hla.rti.CouldNotOpenFED: Module not found serial:0
	at certi.rti.impl.CertiRtiAmbassador.translateException(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.processRequest(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.createFederationExecution(Unknown Source)
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:734)
	at ptolemy.actor.CompositeActor.initialize(CompositeActor.java:912)
	at ptolemy.actor.Manager.initialize(Manager.java:724)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.lib.hoc.ModelReference$1.run(ModelReference.java:626)
java.lang.RuntimeException: ptolemy.kernel.util.IllegalActionException: RTIG could not open FED file: MultiDataTypes.fed
  in .MultiDataTypesConsumer.HlaManager
Because:
Module not found
	at ptolemy.util.MessageHandler.error(MessageHandler.java:115)
	at ptolemy.actor.lib.hoc.ModelReference.executionError(ModelReference.java:450)
	at ptolemy.actor.Manager$ErrorReportingThread.run(Manager.java:1890)
Caused by: ptolemy.kernel.util.IllegalActionException: RTIG could not open FED file: MultiDataTypes.fed
  in .MultiDataTypesConsumer.HlaManager
Because:
Module not found
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:747)
	at ptolemy.actor.CompositeActor.initialize(CompositeActor.java:912)
	at ptolemy.actor.Manager.initialize(Manager.java:724)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.lib.hoc.ModelReference$1.run(ModelReference.java:626)
Caused by: hla.rti.CouldNotOpenFED: Module not found serial:0
	at certi.rti.impl.CertiRtiAmbassador.translateException(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.processRequest(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.createFederationExecution(Unknown Source)
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:734)
	... 4 more
java.lang.RuntimeException: Execution failed
	at ptolemy.moml.MoMLSimpleApplication$UnloadThread.run(MoMLSimpleApplication.java:402)
Caused by: ptolemy.kernel.util.IllegalActionException: Execution failed.
  in .3MultiDataTypes.Producer
Because:
RTIG could not open FED file: MultiDataTypes.fed
  in .MultiDataTypesProducer.HlaManager
Because:
Module not found
	at ptolemy.actor.lib.hoc.ModelReference.fire(ModelReference.java:588)
	at ptolemy.vergil.actor.lib.VisualModelReference.fire(VisualModelReference.java:414)
	at ptolemy.actor.AtomicActor.iterate(AtomicActor.java:512)
	at ptolemy.actor.Director.fire(Director.java:360)
	at ptolemy.actor.CompositeActor.fire(CompositeActor.java:455)
	at ptolemy.actor.Manager.iterate(Manager.java:851)
	at ptolemy.actor.Manager.execute(Manager.java:369)
	at ptolemy.actor.Manager.run(Manager.java:1263)
	at ptolemy.actor.Manager$PtolemyRunThread.run(Manager.java:1915)
Caused by: ptolemy.kernel.util.IllegalActionException: RTIG could not open FED file: MultiDataTypes.fed
  in .MultiDataTypesProducer.HlaManager
Because:
Module not found
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:747)
	at ptolemy.actor.CompositeActor.initialize(CompositeActor.java:912)
	at ptolemy.actor.Manager.initialize(Manager.java:724)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.lib.hoc.ModelReference.fire(ModelReference.java:586)
	... 8 more
Caused by: hla.rti.CouldNotOpenFED: Module not found serial:0
	at certi.rti.impl.CertiRtiAmbassador.translateException(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.processRequest(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.createFederationExecution(Unknown Source)
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:734)
	... 12 more
java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at ptolemy.util.test.junit.AutoKnownFailedTests.RunModel(AutoKnownFailedTests.java:105)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at junitparams.internal.InvokeParameterisedMethod.evaluate(InvokeParameterisedMethod.java:102)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
Caused by: ptolemy.kernel.util.IllegalActionException: Execution failed.
  in .3MultiDataTypes.Producer
Because:
RTIG could not open FED file: MultiDataTypes.fed
  in .MultiDataTypesProducer.HlaManager
Because:
Module not found
	at ptolemy.actor.lib.hoc.ModelReference.fire(ModelReference.java:588)
	at ptolemy.vergil.actor.lib.VisualModelReference.fire(VisualModelReference.java:414)
	at ptolemy.actor.AtomicActor.iterate(AtomicActor.java:512)
	at ptolemy.actor.Director.fire(Director.java:360)
	at ptolemy.actor.CompositeActor.fire(CompositeActor.java:455)
	at ptolemy.actor.Manager.iterate(Manager.java:851)
	at ptolemy.actor.Manager.execute(Manager.java:369)
	at ptolemy.actor.Manager.run(Manager.java:1263)
	at ptolemy.actor.Manager$PtolemyRunThread.run(Manager.java:1915)
Caused by: ptolemy.kernel.util.IllegalActionException: RTIG could not open FED file: MultiDataTypes.fed
  in .MultiDataTypesProducer.HlaManager
Because:
Module not found
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:747)
	at ptolemy.actor.CompositeActor.initialize(CompositeActor.java:912)
	at ptolemy.actor.Manager.initialize(Manager.java:724)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.lib.hoc.ModelReference.fire(ModelReference.java:586)
	... 8 more
Caused by: hla.rti.CouldNotOpenFED: Module not found serial:0
	at certi.rti.impl.CertiRtiAmbassador.translateException(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.processRequest(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.createFederationExecution(Unknown Source)
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:734)
	... 12 more
java.lang.RuntimeException: Execution failed
	at ptolemy.moml.MoMLSimpleApplication$UnloadThread.run(MoMLSimpleApplication.java:402)
Caused by: ptolemy.kernel.util.IllegalActionException: Execution failed.
  in .MultiDataTypes.producer
Because:
RTIG could not open FED file: MultiDataTypes.fed
  in .MultiDataTypesProducer.HlaManager
Because:
Module not found
	at ptolemy.actor.lib.hoc.ModelReference.fire(ModelReference.java:588)
	at ptolemy.vergil.actor.lib.VisualModelReference.fire(VisualModelReference.java:414)
	at ptolemy.actor.AtomicActor.iterate(AtomicActor.java:512)
	at ptolemy.actor.Director.fire(Director.java:360)
	at ptolemy.actor.CompositeActor.fire(CompositeActor.java:455)
	at ptolemy.actor.Manager.iterate(Manager.java:851)
	at ptolemy.actor.Manager.execute(Manager.java:369)
	at ptolemy.actor.Manager.run(Manager.java:1263)
	at ptolemy.actor.Manager$PtolemyRunThread.run(Manager.java:1915)
Caused by: ptolemy.kernel.util.IllegalActionException: RTIG could not open FED file: MultiDataTypes.fed
  in .MultiDataTypesProducer.HlaManager
Because:
Module not found
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:747)
	at ptolemy.actor.CompositeActor.initialize(CompositeActor.java:912)
	at ptolemy.actor.Manager.initialize(Manager.java:724)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.lib.hoc.ModelReference.fire(ModelReference.java:586)
	... 8 more
Caused by: hla.rti.CouldNotOpenFED: Module not found serial:0
	at certi.rti.impl.CertiRtiAmbassador.translateException(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.processRequest(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.createFederationExecution(Unknown Source)
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:734)
	... 12 more
java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at ptolemy.util.test.junit.AutoKnownFailedTests.RunModel(AutoKnownFailedTests.java:105)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at junitparams.internal.InvokeParameterisedMethod.evaluate(InvokeParameterisedMethod.java:102)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
Caused by: ptolemy.kernel.util.IllegalActionException: Execution failed.
  in .MultiDataTypes.producer
Because:
RTIG could not open FED file: MultiDataTypes.fed
  in .MultiDataTypesProducer.HlaManager
Because:
Module not found
	at ptolemy.actor.lib.hoc.ModelReference.fire(ModelReference.java:588)
	at ptolemy.vergil.actor.lib.VisualModelReference.fire(VisualModelReference.java:414)
	at ptolemy.actor.AtomicActor.iterate(AtomicActor.java:512)
	at ptolemy.actor.Director.fire(Director.java:360)
	at ptolemy.actor.CompositeActor.fire(CompositeActor.java:455)
	at ptolemy.actor.Manager.iterate(Manager.java:851)
	at ptolemy.actor.Manager.execute(Manager.java:369)
	at ptolemy.actor.Manager.run(Manager.java:1263)
	at ptolemy.actor.Manager$PtolemyRunThread.run(Manager.java:1915)
Caused by: ptolemy.kernel.util.IllegalActionException: RTIG could not open FED file: MultiDataTypes.fed
  in .MultiDataTypesProducer.HlaManager
Because:
Module not found
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:747)
	at ptolemy.actor.CompositeActor.initialize(CompositeActor.java:912)
	at ptolemy.actor.Manager.initialize(Manager.java:724)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.lib.hoc.ModelReference.fire(ModelReference.java:586)
	... 8 more
Caused by: hla.rti.CouldNotOpenFED: Module not found serial:0
	at certi.rti.impl.CertiRtiAmbassador.translateException(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.processRequest(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.createFederationExecution(Unknown Source)
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:734)
	... 12 more
ptolemy.kernel.util.IllegalActionException: RTIG could not open FED file: MultiDataTypes.fed
  in .MultiDataTypesConsumer.HlaManager
Because:
Module not found
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:747)
	at ptolemy.actor.CompositeActor.initialize(CompositeActor.java:912)
	at ptolemy.actor.Manager.initialize(Manager.java:724)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.lib.hoc.ModelReference$1.run(ModelReference.java:626)
Caused by: hla.rti.CouldNotOpenFED: Module not found serial:0
	at certi.rti.impl.CertiRtiAmbassador.translateException(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.processRequest(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.createFederationExecution(Unknown Source)
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:734)
	... 4 more
Caused by: hla.rti.CouldNotOpenFED: Module not found serial:0
	at certi.rti.impl.CertiRtiAmbassador.translateException(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.processRequest(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.createFederationExecution(Unknown Source)
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:734)
	at ptolemy.actor.CompositeActor.initialize(CompositeActor.java:912)
	at ptolemy.actor.Manager.initialize(Manager.java:724)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.lib.hoc.ModelReference$1.run(ModelReference.java:626)
java.lang.RuntimeException: ptolemy.kernel.util.IllegalActionException: RTIG could not open FED file: MultiDataTypes.fed
  in .MultiDataTypesConsumer.HlaManager
Because:
Module not found
	at ptolemy.util.MessageHandler.error(MessageHandler.java:115)
	at ptolemy.actor.lib.hoc.ModelReference.executionError(ModelReference.java:450)
	at ptolemy.actor.Manager$ErrorReportingThread.run(Manager.java:1890)
Caused by: ptolemy.kernel.util.IllegalActionException: RTIG could not open FED file: MultiDataTypes.fed
  in .MultiDataTypesConsumer.HlaManager
Because:
Module not found
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:747)
	at ptolemy.actor.CompositeActor.initialize(CompositeActor.java:912)
	at ptolemy.actor.Manager.initialize(Manager.java:724)
	at ptolemy.actor.Manager.execute(Manager.java:356)
	at ptolemy.actor.lib.hoc.ModelReference$1.run(ModelReference.java:626)
Caused by: hla.rti.CouldNotOpenFED: Module not found serial:0
	at certi.rti.impl.CertiRtiAmbassador.translateException(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.processRequest(Unknown Source)
	at certi.rti.impl.CertiRtiAmbassador.createFederationExecution(Unknown Source)
	at org.hlacerti.lib.HlaManager.initialize(HlaManager.java:734)
	... 4 more
</pre>
</body>
</html>
